### PR TITLE
Stage one of moving app settings to app.config

### PR DESCRIPTION
**Pull request by adoublebarrel**
### What is the context of this PR?

[Trello Card](https://trello.com/c/4kjDSdfM/1049-move-app-config-from-settings-py-to-flask-application-factory)

Removed settings from new_relic and jwt_decoder, plus associated
changes for their dependents.

### How to review 
Functionality should be unchanged, imports of settings.py should be reduced, config should be being accessed via the Flask applications config dictionary.

### CODE REVIEW

File: tests/app/authentication/test_jwe.py
Comment (collisdigital) [inline tests/app/authentication/test_jwe.py:24]: Nice clean up 👍

File: tests/app/authentication/test_jwe.py
Comment (collisdigital) [inline tests/app/authentication/test_jwe.py]: Not sure why this was originally declared directly here; might make sense to move it into a 'TEST_DO_NOT_USE_SR_PRIVATE_PEM_PASSWORD'?

File: tests/app/authentication/test_jwt.py
Comment (collisdigital) [inline tests/app/authentication/test_jwt.py]: Same as above

File: tests/app/authentication/test_jwt_decoder.py
Comment (collisdigital) [inline tests/app/authentication/test_jwt_decoder.py]: And again

File: tests/app/authentication/test_jwt_decoder_using_rfc_example.py
Comment (collisdigital) [inline tests/app/authentication/test_jwt_decoder_using_rfc_example.py]: Here

File: app/__init__.py
Comment (jonnyshaw89) [inline app/__init__.py]: Would it make sent to make `EQ_MAX_HTTP_POST_CONTENT_LENGTH` `MAX_CONTENT_LENGTH` so that this doesn't need to be set?

**Review by jonnyshaw89 — CHANGES_REQUESTED** [review]

File: app/__init__.py
Comment (adoublebarrel) [reply app/__init__.py]: That occurred to me too, as it's seconded I'll make the change now

**Review by jonnyshaw89 — APPROVED** [review]

**Comment by jonnyshaw89** [comment]
@adoublebarrel FYI we normally need 2 approvals to merge.

**Comment by collisdigital** [comment]
@adoublebarrel also usually not the author who clicks the merge button but admitted by convention rather than standard; we should update the Dev workflow to explain or agree it isn't important?

**Comment by adoublebarrel** [comment]
The merge button protocol is mentioned in the workflow, this was generally my bad. I was keen to get working on the next set of changes, saw green and didn't properly reference the Dev workflow doc. Lessons learnt and I agree with the workflow as it is.

**Pull request merged**
